core-bundle/src/Routing/Matcher/LanguageFilter.php

```diff
@@ -51,6 +51,7 @@ public function filter(RouteCollection $collection, Request $request): RouteColl
                 !$pageModel instanceof PageModel
                 || $pageModel->rootIsFallback
                 || \in_array($pageModel->rootLanguage, $languages, true)
```

Comment:
Do we need this duplicate line? The root language should not be saved with `_`, because backend does only allow values with dash!

Reply:
No, I have actually removed this line in my suggestion (or rather replaced it with `\in_array(str_replace('-', '_', $pageModel->rootLanguage), $languages, true)`). Maybe GitHub has not merged it correctly.